### DISCUSSION TITLE
请问Laravel Octane中应该如何使用，微信支付目前总是提示Yansongda\Pay\Exception\ServiceNotFoundException

### DISCUSSION THREAD

**Discussion by warmup413** [yansongda/pay#721] [Q&A]
已解决

**Comment by luolaihui110** [comment]
怎么处理的呀

**Comment by czhou** [comment]
> 已解决

请问是怎么处理的，我这边也是一直提示 Yansongda\Pay\Exception\ServiceNotFoundException: wechat

**Comment by luolaihui110** [comment]
原issue地址几多 我得看一下



在 2023-09-07 11:57:24，"Charles Zhou" ***@***.***> 写道：

已解决

请问是怎么处理的，我这边也是一直提示 Yansongda\Pay\Exception\ServiceNotFoundException: wechat

**Comment by luolaihui110** [comment]
我需要知道原问题地址，许久都搞忘了，反正 这个我解决了的



在 2023-09-07 11:57:24，"Charles Zhou" ***@***.***> 写道：

已解决

请问是怎么处理的，我这边也是一直提示 Yansongda\Pay\Exception\ServiceNotFoundException: wechat
